com.unity.render-pipelines.high-definition/Documentation~/Override-Screen-Space-GI.md

```diff
@@ -0,0 +1,28 @@
+# Screen Space Global Illumination (Preview)
+
+The **Screen Space Illumination** (SSGI) override is a High Definition Render Pipeline (HDRP) feature that uses the depth and color buffer of the screen to calculate diffuse light bounce.
+
+## Using Screen Space Global Illumination
+
+To use SSGI in your Scene, you must enable it in your [HDRP Asset](HDRP-Asset.md), go to  **Lighting section** and enable the **Screen Space Global Illumination** checkbox.
```

Comment:
Hey, @JordanL8  need your help on this one.

We need to say that the settings must be enabled in HDRP asset.
But we must also say that then it need to be enabled on the camera frame settings (it is by default, but swe must mention it),  and there is the story with Default Settings.

Hierarchy is:
- be sure it is enabled in hdrp settings
- If you are overriding framesettings in camera then check ... else need to look at in Default settings and be sure it is enabled

We should have some kind of share section for those explanation as it apply to a lot of settings, not sure how to do this.

Here SSGI and SSR for example should have the same descirption. THanks